src/pages/payment-methods/_expanded-list.js

```diff
@@ -91,12 +91,15 @@ const Withdrawal = styled(Td)`
     }
 `
 
-const ExpandList = ({ data }) => {
+const ExpandList = ({ data, config, is_crypto }) => {
     const [is_expanded, setIsExpanded] = React.useState(false)
 
     const toggleExpand = () => {
         setIsExpanded(!is_expanded)
     }
+    function getCryptoConfig(name) {
+        return config == undefined ? null : config[name].minimum_withdrawal
```

Comment:
```suggestion
        return typeof config !== undefined && config[name].minimum_withdrawal
```

Reply:
done :+1:

Reply:
i see, how about the condition, is it trying to check for config type or the config itself?

Reply:
config itself.
basically we are initiating the `config` with the `undefined` and we are checking that.

Reply:
ah, no wonder, okay 👍